File: JAVA/Slide_TP/1_Base_de_Java/Tableaux.java
// Écrivez un programme qui crée un tableau d'entiers, le remplit avec des nombres aléatoires, puis affiche ces nombres.

import java.util.Random;

class Tableaux {
    public static void main(String[] args) {
        System.out.println("Bienvenue dans les tableaux!");
        System.out.println("Entrez la taille du tableau:");
        int taille = Integer.parseInt(System.console().readLine());
        int[] tab = new int[taille];
        for(int i=0; i<taille; i++) {
            Random rand = new Random();
            tab[i] = rand.nextInt(100);
            System.out.println("Le " + i+ "eme nombre est " + tab[i]);
        }
    }
}

File: CPP/TP1/Tableaux.cpp
#include <iostream>

using std::cout;
using std::endl;

const int TAILLE = 10;

int main(int, char **) {
    int tab[TAILLE];

    for(int i=0; i<TAILLE; ++i) {
        tab[i] = i % 2;
        cout << tab[i] << " "; 
    }

    cout << endl;

    return TAILLE;
}